pennylane_lightning/core/_serialize.py

```diff
@@ -404,7 +404,7 @@ def get_wires(operation, single_op):
                 uses_stateprep = True
                 continue
             if isinstance(operation, Rot):
-                op_list = operation.expand().operations
+                op_list = operation.decomposition()
```

Comment:
Is it still using the same logic as `expand()` under the hood? What are the performance implications with using `decomposition` instead of `expand`?

Reply:
`Operator.expand` calls `Operator.decomposition` under the hood and package the decomposed operations into a tape. In this case, creating a tape with a list of operations just to retrieve these operations from the tape is quite redundant. Using `decomposition` does literally the same thing.